### DISCUSSION TITLE
Suffering with recognitions.

### DISCUSSION THREAD

**Discussion by juicees** [tphakala/birdnet-go#315] [Q&A]
I've got the device up and running and everything looks reasonably good. Rasperry4/LTE-modem with UgreenUSB/AOM5024-mic installed into ventilated waterproof-box, ready to be installed to the wild. Thanks for the project, cant wait to look forward.

I had faced some challenges with test-identification(indoor) of some common species. Analyzer is not giving any results when playing sounds from phone, example Coal Tit, Willow tit and some another species which are still fairly common in southern Finland. Birdnet-android app does recognize those easily from same soundsource.

I dont believe i have any problems with mic/audio itself, all saved clips from earlier recognizons comes clear sound without any buzzling sounds or so. Also did try to disable all filters and coordinates etc and did change sens-threshold.

Is the BirdNET audio model so much different from Android-app in these days or is there maybe something i can still improve? Like, i still didint figure out how to see active-log of analyzers live session, example what percent of confidence it does get from my test-sounds or some hint from something.

**Comment by tphakala** [comment]
Hi, did you manage to solve detection issues as this discussion is set to closed?

**Reply by juicees** [reply]
Hi. I was actually closing it for a while, cause i DID notice shortly after my post that my device did not recognize anymore basicly anything expect some specific(familiar from past..?) species. I was deciding to install everything again to new os(raspi-lite) and take a look if problem solves by itself. Sorry for my lack of competence with github, im new in this platform.

**Reply by tphakala** [reply]
ok, no worries. Which version of BirdNET-Go are you using?
I would recommend using latest docker image for best results, install with "docker pull ghcr.io/tphakala/birdnet-go:dev".

**Reply by juicees** [reply]
Hey! I had been always using :dev, yes. Now everything is working really well, was right move to close this topic in time. Was just reinstalling everything and now i have all working again. Next step will be to activate LTE-stick to throw my box into field(online with ngrok tunnel / webterminal.) This seems to be amazing project! Would love to innovate a way to share project with Finnish municipalities to migrate this part of their nature-services to strengthen residents nature awareness.
